README.md

```diff
@@ -23,11 +23,18 @@ Pretty printing is very much needed for nested classes like trees.
 
 Using the verbose argument is nice, since it makes it easy to switch between whether to print or not. 
 
+Invariants of functions are important 
+
+
 ## Problems encountered if any.
 
 For the naive, we had problems with splitting edges/nodes. This was solved by pen and paper, going sytematicly through cases, and using pretty printing. 
 
-As mentioned above, we missed an edge case for McCreight’s algorithm, which resulted in it being half-way done in this version of the handin. 
+Initially we missed an edge case for McCreight’s algorithm, which resulted in it having the same worst-case running time as the naive. 
+
+McCreight’s algorithm was difficult to get right, mostly since the indecies where difficult. 
+
+The switch from a group of two to a group of one, changed the work process. 
```

Comment:
Is this not a hand-in for both of you? (I need to know such things for grading, so I want to hear more about that if that is the case...)

Reply:
Technically this one is for both of us, but it doesn't really matter.
Jens got a job and is dropping out, so I made the last part of the handin myself.

Reply:
Ok, I'll keep that in mind. Cheers